### ISSUE TITLE
Contact Font Saturation Needs To Be Fixed Within 

### ISSUE THREAD

**Issue by therealslimhsiehdy** [cisagov/manage.get.gov#2750]
### Current Behavior

Currently when we have more than one Contact in the "Other employees from your organization" in both the review page when you submit a domain request, and when you have a domain request in review state it displays as this grey font that is almost transparent with a white background.

See screenshot below for what it currently looks like 

### Expected Behavior

Contacts are in [this Figma spec here](https://www.figma.com/design/xFtGLHVrhp0lvwh0gYWnbx/.gov-registar?node-id=2703-31376&node-type=canvas&t=Sd5vKuWCqh2BIwoO-0) so it's viewable.

<img width="782" alt="Screenshot 2024-09-19 at 3 53 59 PM" src="https://github.com/user-attachments/assets/c42f13b5-ab61-479e-8d8b-cf475d6e374e">

### Steps to Reproduce

1. Create a domain request and in the "Other employees from your organization" section, add more than one contact. When you get the last "review" page part, you'll see the font is wonky
2. Same for if a domain request is in the "in review" state and you have more than 1 contact 

### Additional Context

<img width="701" alt="Screenshot 2024-09-13 at 8 42 39 AM" src="https://github.com/user-attachments/assets/878d1e10-9290-41a5-bea1-b3eab1812c77">
<img width="572" alt="Screenshot 2024-09-13 at 8 44 31 AM" src="https://github.com/user-attachments/assets/8a06722e-0923-4b6c-9bb3-bbc02fa4b36d">

**Comment by abroddrick** [comment]
marking as bug severity of high, because it is an obvious bug in addition to being an accessibility issue

**Comment by abroddrick** [comment]
I added the accessibility tag, but @katypies or @PaulKuykendall is there any other thing we should be putting on on accessibility?
